tests/test_code_report.py

```diff
@@ -405,3 +405,31 @@ def test_code_report_extended_arg_search_embedded(tmp_path: pathlib.Path, stdout
 
     env.run()
     stdout_checker.assert_absent_calls_with_param("${CODE_REPORT_FILE}")
+
+
+@pytest.mark.parametrize('analyzer, extra_args, tested_content', [
+    ['clang_format', ["--report-html"], source_code_c],
```

Comment:
You are not checking html file after running clang_format, so you don't need to add the `--report-html` parameter.
BTW, the html generation is already checked in another test.

Reply:
Thank you, fixed